springwolf-core/src/main/java/io/github/springwolf/core/asyncapi/schemas/SwaggerSchemaService.java

```diff
@@ -184,6 +185,14 @@ private void processAsyncApiPayloadAnnotation(Map<String, Schema> schemas, Strin
         }
     }
 
+    private void processSchemaWithoutName(Map<String, Schema> schemas) {
+        schemas.forEach((schemaName, schema) -> {
+            if (schema.getName() == null) {
+                schema.setName(schemaName);
```

Comment:
I am wondering, whether this is the better fix, since the Map from swagger contains the correct class name, but it is not set in the schema#name

At the same time, I am unsure, whether we should touch/modify the schema for this?